scripts/mppc_to_multiprobe.py

```diff
@@ -0,0 +1,298 @@
+        Increase the scan size (normally 3.2um) to cover the active area of the full mppc detector.
+        Set 2000x magnification and 8V at SEM.
+        Set beam shift to zero.
+        Select single-beam mode; use beamlet E4.
+        Set the scan mode to ‘External’.
+        All mppc cells should be active.
+    For scan control:
+        Set galvo mirror deflection to zero; X = Y = 0 (descan).
+    Configure the camera to acquire a single field image.
+
+The scanning is performed with one beamlet/single probe E4 using all mppc cells.
+
+Correction on hardware:
+    The offset correction needs to be applied to the galvo mirror settings.  # TODO to be defined!
+    The rotation correction needs to be manually adjusted via the rotational mounting on the mppc detector.
+
+TODO extend this description after testing and when workflow is more specified.
+"""
+
+from __future__ import division
+
+import argparse
+import logging
+import math
+import numpy
+
+import cv2
+import glob
+import os
+
+import skimage
+
+from odemis.util.img import RGB2Greyscale
+
+from odemis.dataio import tiff
+from libtiff import TIFF
+from odemis.util import almost_equal
+import sys
+from scipy.ndimage.measurements import center_of_mass
+
+
+def preprocess_images(image):
+    """
+    Pre-processing the data to calculate the corrections. The size of the input image is well defined (6400px x 6400px).
+    The image is acquired with one beamlet (E4/(5,4)) and contains 64 cell images (8x8; 800px x 800px; 30um x 30um).
+    Each cell image shows the corresponding active mppc cell image as a bright square. All other cells are inactive
+    and appear dark in the single cell image.
+    The single cell image covers a field of view larger than all mppc cells.
+    The mppc cell image located in the single cell image (bright square) consists of a fixed size of about
+    3um x 3um (80px x 80px), which corresponds to about 1% of the area of a single cell image.
+    Depending on which of the 64 cell images, the bright square is located in a different position of the image.
+    The 64 cell images are stitched together. A projection of all single cell images onto each other would result
+    in one image with all mppc cell images being bright. As the image is recorded using the E4 beamlet,
+    the E4 cell image should show the mppc cell image at the center of the image if correctly aligned.
+    The corner images A1, A8, H1 and H8 are used to correct for rotation, the E4 image is used to correct for an offset.
+    :param image: (DataArray) Image of size 6400px x 6400px made up by 64 cell images (800px x 800px), each containing
+                the corresponding mppc cell image (bright square) at a given position, which are shifted by the size
+                of the square in respect to each other.
+    :returns: (list of 4 ndarrays) The preprocessed corner images. Shape is (y, x).
+              (ndarray) The E4 cell image. Shape is (y, x).
```

Comment:
Let's be a little bit more specific on the data type. It's an uint8, which contains 0 for the "background" and a number (typically 1, but not necessarily) for the area. Right?